mcux/components/CMakeLists.txt

```diff
@@ -6,4 +6,11 @@ if(CONFIG_ETH_MCUX)
   zephyr_include_directories(phyksz8081)
 endif()
 
+if(CONFIG_SDMMC_USDHC_PMIC_1_8_VOL)
```

Comment:
Can we add a more generic config to include the PMIC PCA9420 driver. This is in case future Zephyr drivers need this component. One suggestion for a name is `CONFIG_USE_PMIC_PCA9420`